src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -1,12 +1,16 @@
 import { ChartType, Plugin } from "chart.js";
-import { chartFontColor } from "../../../../helpers/figures/charts/chart_common";
+import {
+  TREND_LINE_XAXIS_ID,
+  chartFontColor,
+} from "../../../../helpers/figures/charts/chart_common";
 import { Color } from "../../../../types";
 
 interface ChartShowValuesPluginOptions {
   showValues: boolean;
   background?: Color;
   horizontal?: boolean;
   callback: (value: number | string) => string;
+  ignoredDataSets?: number[];
```

Comment:
I guess this is a left over from a previous implementation?

Reply:
yup ! and it's updated